src/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"html/template"
+	"io"
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+//Template ......
+type Template struct {
+	templates *template.Template
+}
+
+func main() {
+	e := echo.New()
+
+	t := &Template{
+		templates: template.Must(template.ParseGlob("templates/*.html")),
+	}
+	e.Renderer = t
+
+	e.GET("/", serveLanding)
```

Comment:
Consider separating out routing into a function - although this will do for now.